awx/api/generics.py

```diff
@@ -964,14 +966,19 @@ def post(self, request, *args, **kwargs):
         if hasattr(new_obj, 'admin_role') and request.user not in new_obj.admin_role.members.all():
             new_obj.admin_role.members.add(request.user)
         if sub_objs:
+            # store the copied object dict into memcached, because it's
+            # often too large for postgres' notification bus
+            # (which has a default maximum message size of 8k)
+            key = 'deep-copy-{}'.format(str(uuid.uuid4()))
+            cache.set(key, sub_objs, timeout=3600)
```

Comment:
I also set a sane-ish 1 hour timeout just in case the worker queue gets backed up, so these don't pile up and consume lots of memory